frame/staking/src/pallet/impls.rs

```diff
@@ -974,6 +974,46 @@ impl<T: Config> Pallet<T> {
 }
 
 impl<T: Config> Pallet<T> {
+	/// Returns the current staking reward rate.
+	pub fn api_reward_rate(
```

Comment:
Staking is currently unaware of the reward rate, this implementation makes assumption on the implementation of `Config` trait.

The payout is abstracted through associated type `Config::EraPayout` which bounds the trait `EraPayout`
Either some assumption should be enforced on the implementation of `EraPayout` on the rate can be retrieved by calling the payout for an era of one year `payout(current_total_staked, current_total_issuance, one_year)`.
But this doesn't look so good.

Better would be to:
* either add a method to `EraPayout` trait to be able to retrieve the reward rate
* or expose the rate from the node.

Reply:
Indeed. The calculations currently in place were based on an outdated calculation of PJS. We can simply refer to era payout to determine the inflation on a per-era basis.